internal/backend/db/dbgorm/sessions_test.go

```diff
@@ -64,12 +65,18 @@ func testLastLogRecord(t *testing.T, f *dbFixture, numRecords int, sessionID sdk
 	assert.Equal(t, l, lr)
 }
 
+func assertSessionLogRecordsEqual(t *testing.T, a, b sdktypes.SessionLogRecord) {
+	a = a.WithProcessID(fixtures.ProcessID()).WithoutTimestamp()
+	b = b.WithProcessID(fixtures.ProcessID()).WithoutTimestamp()
+	assert.Equal(t, a, b)
+}
+
 func TestCreateSession(t *testing.T) {
 	f := preSessionTest(t)
 
 	// test createSession
 	s := f.newSession(sdktypes.SessionStateTypeCompleted)
-	f.createSessionsAndAssert(t, s) // all session assets are optional and will be set to nil
+	f.createSessionsAndAssert(t, s) // all session asserts are optional and will be set to nil
```

Comment:
assets :) not asserts. e.g. build, env, deployment, event

Reply:
fixed